tests/test_move.py

```diff
@@ -37,6 +38,17 @@ def test_move_fs(self):
             self.assertEqual(dst_file1_info.modified, src_file1_info.modified)
             self.assertEqual(dst_file2_info.modified, src_file2_info.modified)
 
+    def test_move_file(self):
+        with open_fs("temp://") as temp:
+            syspath = temp.getsyspath("/")
+            a = open_fs(syspath)
+            a.makedir("dir")
+            b = open_fs(join(syspath, "dir"))
+            b.writetext("file.txt", "Content")
+            fs.move.move_file(b, "file.txt", a, "here.txt")
```

Comment:
Since src_fs and dst_fs can be FS objects or FS-URLs, perhaps it's worth testing passing the FSes by FS-URL also? :shrug:  (or perhaps that's unnecessary? @althonos ? )

Reply:
I missed that - I'll add a test for that.